backend/server/server.js

```diff
@@ -3,6 +3,7 @@ import http from 'http';
 import path from 'path';
 import url from 'url';
 import { parse } from 'querystring';
+import Loadable from 'react-loadable';
```

Comment:
So, dynamic imports OTB from CRA doesn't support server side (as per doc) I had to use this: https://github.com/jamiebuilds/react-loadable